diff --git a/dist/index.d.cts b/dist/index.d.cts
index dea8c54990d7dea6428bd00b67d20d62419224ab..dc17ae332bc36d0dd6b62edcd6a69a7e0d4d3d1b 100644
--- a/dist/index.d.cts
+++ b/dist/index.d.cts
@@ -256,6 +256,7 @@ declare const createRoute: <P extends string, R extends Omit<RouteConfig, "path"
 }>(routeConfig: R) => R & {
   getRoutingPath(): RoutingPath<R["path"]>;
 };
+import * as zodModule from "zod";
 import z = zodModule.z;
 //#endregion
 export { $, DeepSimplify, HonoToOpenAPIHono, Hook, MiddlewareToHandlerType, OfHandlerType, OpenAPIGeneratorConfigure, OpenAPIGeneratorOptions, OpenAPIHono, OpenAPIHonoOptions, OpenAPIObjectConfigure, OpenAPIRoute, RouteConfig, RouteConfigToEnv, RouteConfigToTypedResponse, RouteHandler, RouteHook, createRoute, defineOpenAPIRoute, extendZodWithOpenApi, z };
diff --git a/dist/index.d.mts b/dist/index.d.mts
index dd4ce23279634ecb8ce098d957ead64ab10d3f3e..7226d5ebefd57c568d45b6eb484b1363269dad7c 100644
--- a/dist/index.d.mts
+++ b/dist/index.d.mts
@@ -256,6 +256,7 @@ declare const createRoute: <P extends string, R extends Omit<RouteConfig, "path"
 }>(routeConfig: R) => R & {
   getRoutingPath(): RoutingPath<R["path"]>;
 };
+import * as zodModule from "zod";
 import z = zodModule.z;
 //#endregion
 export { $, DeepSimplify, HonoToOpenAPIHono, Hook, MiddlewareToHandlerType, OfHandlerType, OpenAPIGeneratorConfigure, OpenAPIGeneratorOptions, OpenAPIHono, OpenAPIHonoOptions, OpenAPIObjectConfigure, OpenAPIRoute, RouteConfig, RouteConfigToEnv, RouteConfigToTypedResponse, RouteHandler, RouteHook, createRoute, defineOpenAPIRoute, extendZodWithOpenApi, z };
